server/sonar-plugin-bridge/src/main/java/org/sonar/plugin/PrivilegedPluginBridge.java

```diff
@@ -17,28 +17,30 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonar.server.governance;
+package org.sonar.plugin;
 
 import org.sonar.core.platform.ComponentContainer;
 
 /**
- * Interface implemented by the Extension point exposed by the Governance plugin that serves as the unique access point from
- * the whole SQ instance into the Governance plugin.
+ * Interface implemented by the Extension point exposed by the Privileged plugin that serves as the unique access
+ * point from the whole SQ instance into the Privileged plugin.
  */
-public interface GovernanceBridge {
+public interface PrivilegedPluginBridge {
+
+  public String getPluginName();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) "public" is redundant in this context. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2333)